python/lsst/pipe/tasks/maskStreaks.py

```diff
@@ -378,6 +378,7 @@ def fit(self, dChi2Tol=0.1, maxIter=100):
             Maximum number of fit iterations allowed. The fit should converge in
             ~10 iterations, depending on the value of dChi2Tol, but this
             maximum provides a backup.
+        log : TODO
```

Comment:
Please complete the TODO 😄

Reply:
I think as I typed TODO I subconsciously knew I would forget it....